### ISSUE TITLE
Typo in twisted/plugins/twisted_lore.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5326]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#5326|
|Type|defect|
|Created|2011-10-21 14:35:08Z|
|Branch|https://github.com/twisted/twisted/tree/lore-plugin-5326|

The MathProcessor plugins specifies twsited.lore.lmath as the target module, which is unlikely to work.

Attachments:

* [5326.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5e1/5e1b52f118d2f1e2db8104bb9f755a0de9337109/f487b22d68694a02f000de6cab9a32e7dab224cc.patch) (406 bytes) - added by moijes12 on 2012-01-19 06:19:39Z - 
* [Twisted_5326.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5e1/5e1b52f118d2f1e2db8104bb9f755a0de9337109/3af6c4f4855950c05cd32c71e6c11e75ef3b23c8.patch) (762 bytes) - added by moijes12 on 2012-02-12 12:49:44Z - twisted.plugins.twisted_lore's MathProcessor plugin now correctly calls twisted.lore.lmath instead of twsited.lore.lmath.

<details><summary>Searchable metadata</summary>

```
trac-id__5326 5326
type__defect defect
reporter__therve therve
priority__low low
milestone__ 
branch__branches_lore_plugin_5326 branches/lore-plugin-5326
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__ 
time__1319207708000000 1319207708000000
changetime__1331664321000000 1331664321000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="moijes12's avatar" src="https://avatars.githubusercontent.com/u/4566851?s=50" width="50" height="50">](https://github.com/moijes12)<a name="note_1"></a>|@moijes12 commented|
|-|-|

Therve : Please remove my attachment. If no new contributor changes it after some time, I'll re-attach the patch.

**Comment by twisted-trac** [comment]
|[<img alt="MostAwesomeDude's avatar" src="https://avatars.githubusercontent.com/u/118035?s=50" width="50" height="50">](https://github.com/MostAwesomeDude)<a name="note_5"></a>|@MostAwesomeDude set owner to @moijes12|
|-|-|

I'm hard-pressed to find a single thing wrong with this patch. :3

Looks good to go. Could somebody with commit access merge this, please?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

One thing missing from the patch is a unit test.  Remember, every single code change to Twisted requires unit test coverage - at least full line and branch coverage.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [33687]) Branching to 'lore-plugin-5326'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

(In [33688]) Apply patch and add a unit test

refs [#5326](https://github.com/twisted/twisted/issues/5326)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun removed owner|
|-|-|

Stuck the patch in a branch, added a test, and moved the lore news fragment to the lore topfiles directory.

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/lore-plugin-5326)

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_10"></a>|@washort set owner to @exarkun|
|-|-|

ftw

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun set status to `closed`|
|-|-|

Fixed in r33739:

    Merge lore-plugin-5326 
    Author: moijes12, exarkun
    Reviewer: exarkun, washort
    Fixes: [#5326](https://github.com/twisted/twisted/issues/5326)

    Fix a typo in the lore math plugin definition which prevented it from actually
    being usable.
